roles/openshift_health_checker/library/aos_version.py

```diff
@@ -47,8 +47,6 @@ def bail(error):  # pylint: disable=missing-docstring
 
     # determine what level of precision we're expecting for the version
     expected_version = module.params['version']
-    if expected_version.startswith('v'):  # v3.3 => 3.3
-        expected_version = expected_version[1:]
```

Comment:
`openshift_release` is used by a lot of other stuff in the installer, I'm just piggy-backing on it here. Actually it's not well-documented what it's supposed to be, and I'm finding it doesn't have to be specified. Worth documenting as you say.